website/blog/2021-01/01/_index.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Happy anniversary Gardener! Three years of open source Kubernetes management
+linkTitle: Happy anniversary Gardener! Three years of open source Kubernetes management
+newsSubtitle: January 30, 2021
+type: blog
+publishdate: 2021-01-25
+archivedate: 2021-02-25
```

Comment:
`archivedate`=`publishdate` means that the news in the news feed on the front page will disappear the moment it's published. Give it some time to stay there.

Reply:
yes, my bad. ignore